src/extensions/cluster-feature.ts

```diff
@@ -44,23 +44,23 @@ export abstract class ClusterFeature {
    * 
    * @param cluster the cluster that the feature is to be installed on
    */
-  abstract async install(cluster: Cluster): Promise<void>;
+  abstract install(cluster: Cluster): Promise<void>;
 
   /**
    * to be implemented in the derived class, this method is typically called by Lens when a user has indicated that this feature is to be upgraded. The implementation
    * of this method should upgrade the kubernetes resources already installed, if relevant to the feature
    * 
    * @param cluster the cluster that the feature is to be upgraded on
    */
-  abstract async upgrade(cluster: Cluster): Promise<void>;
```

Comment:
IDE / `tsc` complains.. or maybe some tsc versions messed up, I've tried to update to latest one at some point.

Reply:
![image](https://user-images.githubusercontent.com/6377066/101072759-2d415900-35a7-11eb-888c-88d06960089b.png)

Reply:
It seems TypeScript has recently forbidden it. The implementation can still use async if wanted.

Reply:
I wonder why @ixrock IDE complains, shouldn't the IDE use the TypeScript compiler specified in package.json? Or did you override in your setting?